lib/qless/middleware/requeue_exceptions.rb

```diff
@@ -53,21 +53,29 @@ def on_requeue_callback
         @on_requeue_callback ||= DEFAULT_ON_REQUEUE_CALLBACK
       end
 
-      def around_perform(job)
-        super
-      rescue *requeueable_exceptions.keys => e
-        config = requeuable_exception_for(e)
+      def exception_handler(job, error)
```

Comment:
WDYT about renaming this to `handle_exception`?  The method name `exception_handler` is a noun, which would suggest that it returns an object that is meant to be used to handle exceptions.  `handle_exception`, OTOH, as a verb is more clear that the method itself handles exceptions.